src/components/modules/contacts/ui/tables/MyContactTable.tsx

```
@@ -0,0 +1,190 @@
+
+  const toggleRowExpansion = (contactId: string) => {
+    setExpandedRows((prev) =>
+      prev.includes(contactId)
+        ? prev.filter((id) => id !== contactId)
+        : [...prev, contactId],
+    );
+  };
+
+  return (
+    <div className="container mx-auto py-3">
+      {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)